### PR TITLE
Add containers and configure environment

### LINKED ISSUE #21: Docker enviroment for develop
Hi, @flyve-mdm/backend-development 

For develop GLPI is required configure an environment in the host machine with all required it. This is a disadvantage by the time necessary in this process.

**Describe the solution you'd like**
Configure containers docker for develop, where everything is ready to work. Use docker compose for configure this enviroment.

**Additional context**
This enviroment must have a web server, database server and PHP as minimun requires.

### PR DESCRIPTION
**Pull request by ingluife**
Hi, @flyve-mdm/backend-development .

This is the configuration of all docker containers and the environment.
I am checking the issue in the git clone and the composer install for GLPI application.

Closes #21

### CODE REVIEW

**Comment by ajsb85** [comment]
Hi, @ingluife 
There are commits without signature and others with signature unverified.

**Review by ajsb85 — CHANGES_REQUESTED** [review]
Hi, @ingluife 
There are commits without signature and others with signature unverified.

**Review by ajsb85 — DISMISSED** [review]

File: .env [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+#Enviroment MYSQL
+MYSQL_TAG=5.6.35
+
+#Enviroment Apache
+APACHE_TAG=2.4
+
+#Enviroment GLPI
+CLONNING_METHOD=ssh
+GLPI_BRANCH=9.2/bugfixes
+GLPI_SOURCE=git@github.com:glpi-project/glpi.git
```
Comment (ajsb85) [inline .env]: Hi, @ingluife 
Maybe a new env var, for the server name: `dev.glpi.local`

File: docker/php/custom.ini [hunk 2]
```diff
@@ -0,0 +1,29 @@
+; General settings
+date.timezone = "America/Caracas"
```
Comment (ajsb85) [inline docker/php/custom.ini]: Hi, @ingluife 
Maybe this time zone can be the default and set up optionally with an external arguments.

**Review by ajsb85 — DISMISSED** [review]

File: docker/php/custom.ini (hunk 2 above)
Comment (ingluife) [reply docker/php/custom.ini]: Ok, perfect. Today i will do it. But default Time Zone should be UTC. Do you agree?

File: .env (hunk 1 above)
Comment (ingluife) [reply .env]: Good, i will add it as environment variable.

**Comment by ingluife** [comment]
Hi @flyve-mdm/backend-development.

This docker environment is ready for tested.

**Review by ajsb85 — APPROVED** [review]

**Pull request merged**